charts/sysdig/values.yaml

```diff
@@ -177,3 +177,23 @@ tolerations:
 prometheus:
   file: false
   yaml: {}
+
+ #extraVolumes: {}
+  # Allow passing extra volumes to the agent to mount secrets or certificates
+  # to authenticate in different services.
+  # Any kind of volume can be passed. Example:
+  #
+  # extraVolumes:
+  #   volumes:
+  #     - name: sysdig-new-cm
+  #       configMap:
+  #         name: my-cm
+  #         optional: true
+  #     - name: sysdig-new-secret
+  #       secret:
+  #         secretName: my-secret
+  #   mounts:
```

Comment:
And this one, because is part of the schema

Reply:
done